Example2/TestCases/test_case2.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * gcc -m32 -static -o test2 test_case2.c
+ * ./test2
+ 
+ */
+#define _GNU_SOURCE
+#include <err.h>
+#include <errno.h>
+#include <sched.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <net/if.h>
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <linux/netfilter_ipv4/ip_tables.h>
+
+int main(int argc, char *argv[]) {
+  int s;
+
+  if (unshare(CLONE_NEWUSER) != 0) err(1, "unshare(CLONE_NEWUSER)");
+  if (unshare(CLONE_NEWNET) != 0) err(1, "unshare(CLONE_NEWNET)");
+
+  if ((s = socket(AF_INET, SOCK_STREAM, 0)) < 0) err(1, "socket");
+
+///////// Different values of size passed in //////////////
+  struct {
+    struct ipt_replace replace;
+    struct ipt_entry entry;
+    struct xt_entry_match match;
+    struct xt_entry_target target;
+  } data = {0};
+
+  data.replace.num_counters = 1;
+  data.replace.num_entries = 1;
+  data.replace.size = 5000; // size changed here
```

Comment:
If you want to test different values passed in, you should make the values changeable instead of hardwired as constant.